Basics/Writing bug reports.md

```diff
@@ -11,110 +11,151 @@ Each bug report, at a minimum, consists of a title and a description.
 ## Title
 
 The title can either be:
-
 - An imperative denoting what needs to be done.
 	- E.g. "Add an Info button to the login screen"
 - Or a description of the error.
 	- E.g. "The app crashes when I take a photo of myself"
 
-## Description
-
-### Header - mobile apps
-
-In the "Device" field - name those devices on which you managed to reproduce the issue.
+## Device/OS/App version info:
 
-In the "Build" field - name the build on which you managed to reproduce the issue. Add the environment when needed.
+The device could be any type of machine that is used during testing.
+E.g. “mobile device, computer, car display, home appliance”
 
-> Device: Nexus 5X (Android 7.0), HTC M9 (Android 6.0)   
-> Build: 1.1.0-b4422 (Development)
+OS could be any piece of software the user is operating at times
+E.g. “some car OS, tv OS, Firmware of appliances if needed, mobile OS, etc.”
 
-### Header - web apps
+App version where bug occurred
```

Comment:
Maybe add here app version (or "build) and environment where bug occured - or keep the old wording - "In the "Build" field - name the build on which you managed to reproduce the issue. Add the environment when needed."